{"name": "test_providing_wrongly_types_query", "status": "broken", "statusDetails": {"message": "ValueError: Single '}' encountered in format string", "trace": "a = (), kw = {}, __tracebackhide__ = True, params = OrderedDict(), args = []\n\n    @wraps(func)\n    def impl(*a, **kw):\n        __tracebackhide__ = True\n        params = func_parameters(func, *a, **kw)\n        args = list(map(lambda x: represent(x), a))\n>       with StepContext(self.title.format(*args, **params), params):\nE       ValueError: Single '}' encountered in format string\n\n../venv/lib/python3.8/site-packages/allure_commons/_allure.py:171: ValueError"}, "description": "Prove that providing a string where an int is expected results in 400 (Bad Request).", "start": 1646730859605, "stop": 1646730859605, "uuid": "12a80da1-7679-4977-8274-1b57a30e7c22", "historyId": "368c27a9039a603f11a828df8574fd9f", "testCaseId": "fdaf31d0b6c02d1954d29efdcd1205ae", "fullName": "e2etest.test_datasetngfz#test_providing_wrongly_types_query", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetngfz"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "47719-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetngfz"}]}